nhrpd/vici.c

```diff
@@ -522,14 +522,18 @@ void vici_terminate(void)
 {
 }
 
-void vici_terminate_vc(unsigned int ike_id)
+void vici_terminate_vc_by_profile_name(char *profile_name)
+{
+	struct vici_conn *vici = &vici_connection;
+	vici_submit_request(vici, "terminate", VICI_KEY_VALUE, "ike",
+		    strlen(profile_name), profile_name, VICI_END);
+}
+
+void vici_terminate_vc_by_ike_id(unsigned int ike_id)
 {
 	struct vici_conn *vici = &vici_connection;
 	char ike_id_str[10]={0};
 	snprintf(ike_id_str, sizeof(ike_id_str), "%d", ike_id);
-	debugf(NHRP_DEBUG_VICI,"ike_id_str = %s", ike_id_str);
```

Comment:
Any reason not to keep extra debug info?

Reply:
@Jafaral I have pushed new commit to add back this debug.